### PR TITLE
test: coveralls + megalinter

### PR DESCRIPTION
**Pull request by amagyar-iohk**
# Overview

Fixes #<issue number>

### My PR contains...
* [ ] No code changes (changes to documentation, CI, metadata, etc.)
* [ ] Bug fixes (non-breaking change which fixes an issue)
* [ ] Improvements (misc. changes to existing features)
* [ ] Features (non-breaking change which adds functionality)

### My changes...
* [ ] are breaking changes
* [ ] are not breaking changes
* [ ] If yes to above: I have updated the documentation accordingly

### Documentation
* [ ] My changes do not require a change to the project documentation
* [ ] My changes require a change to the project documentation
* [ ] If yes to above: I have updated the documentation accordingly

### Tests
* [ ] My changes can not or do not need to be tested
* [ ] My changes can and should be tested by unit and/or integration tests
* [ ] If yes to above: I have added tests to cover my changes
* [ ] If yes to above: I have taken care to cover edge cases in my tests

### CODE REVIEW

File: atala-prism-sdk/build.gradle.kts [hunk 1]
```diff
@@ -55,6 +74,8 @@ kotlin {
     }
     applyDefaultHierarchyTemplate()
 
+    applyDefaultHierarchyTemplate()
+
```
Comment (cristianIOHK) [inline atala-prism-sdk/build.gradle.kts:78]: This is duplicate.

**Review by cristianIOHK — CHANGES_REQUESTED** [review]
Requested a small change, the rest looks fine

File: .mega-linter.yml [hunk 2]
```diff
@@ -1,16 +1,12 @@
 APPLY_FIXES: none
-FILTER_REGEX_EXCLUDE: (karma.config.js|polyfill.js|timeout.js|CHANGELOG.md)
```
Comment (hamada147) [inline .mega-linter.yml:2]: Please add to this line the test/ not just the test/

File: atala-prism-sdk/build.gradle.kts [hunk 3]
```diff
@@ -17,23 +17,42 @@ plugins {
...
+    }
+}
+
 koverReport {
+    filters {
+        excludes {
+            packages(
+                "io.iohk.atala.prism.protos",
+                "io.iohk.atala.prism.walletsdk.domain",
+                "ioiohkatalaprismwalletsdkpluto.data"
```
Comment (hamada147) [inline atala-prism-sdk/build.gradle.kts:34]: This should be like that
```suggestion
                "io.iohk.atala.prism.walletsdk.pluto.data"
```

File: atala-prism-sdk/build.gradle.kts [hunk 4]
```diff
@@ -55,6 +74,8 @@ kotlin {
     }
     applyDefaultHierarchyTemplate()
 
+    applyDefaultHierarchyTemplate()
```
Comment (hamada147) [inline atala-prism-sdk/build.gradle.kts:77]: This should be removed 
```suggestion
```

**Review by hamada147 — CHANGES_REQUESTED** [review]

File: .mega-linter.yml (hunk 2 above)
Comment (amagyar-iohk) [reply .mega-linter.yml:2]: that's not required, it won't validate js stuff, the `test` exclude is for the e2e test

**Review by hamada147 — APPROVED** [review]

**Review by cristianIOHK — APPROVED** [review]
